chaincfg/params.go

```diff
@@ -262,7 +277,7 @@ var MainNetParams = Params{
 	Name:        "mainnet",
 	Net:         wire.MainNet,
 	DefaultPort: "8333",
-	DNSSeeds: []DNSSeed{
+	DNSSeeds: []DNSSeed{ // TODO change this to proper dns seed
```

Comment:
We should definitely fix this to working seeds, however we can keep seed.bchd.cash as this will be fixed once these nodes are deployed.

Reply:
Yeah it doesn't really create an issue so I just kept it for now. Hoping we get seed.bchd.cash back.